packages/tables/src/Concerns/CanPaginateRecords.php

```diff
@@ -15,7 +15,7 @@ trait CanPaginateRecords
 
     public $tableRecordsPerPage;
 
-    protected int $defaultTableRecordsPerPageSelectOption = 10;
+    protected int $defaultTableRecordsPerPageSelectOption = 0;
```

Comment:
```suggestion
    protected ?int $defaultTableRecordsPerPageSelectOption = null;
```

Reply:
Are you sure? This is a breaking change for people who did:

```php
protected int $defaultTableRecordsPerPageSelectOption = 50;
```

Reply:
Doesn't PHP allow this, since `int` is a subset of `?int` ? testing this would be great

Reply:
<img width="1170" alt="Schermafbeelding 2022-05-26 om 22 12 31" src="https://user-images.githubusercontent.com/59207045/170572159-da70b5ee-8eb4-4f6b-9d71-214df47125a8.png">

Reply:
Good idea!